rtd_docs/conf.py

```diff
@@ -132,8 +132,14 @@ def autodoc_process(app, what: str, name: str, obj: Any, options,
 
 
 def source_read(app, docname, source):
-    source[0] = re.sub(r'"##### (Copyright 20\d\d The Cirq Developers)"',
-                       r'"**\1**"', source[0])
+    source[0] = re.sub(r'"##### (Copyright 20\d\d The Cirq Developers)"', r'""',
+                       source[0])
+    source[0] = re.sub(
+        r'(\{\s*?"cell_type": "code".*?"#@title.*License.".*?\},)',
```

Comment:
nit:  Why have the question mark in ".*?"  Doesn't "*" mean zero or more already?

Reply:
`?` is for non-greedy matching - i.e. it is trying to find the shortest possible match for `.*` which is by default greedy.

Reply:
I guess I could remove the `?` from the first one but the second and the third one need it otherwise it would match larger, cross cell strings.

Reply:
Ah, gotcha.  Thanks for the explanation.